### PR TITLE
removes unused configs

### PR DESCRIPTION
**Pull request by esotrope**
followup PR will be issued to change gitlab configs to match

### CODE REVIEW

File: admin-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # actuator end-points should be served from a different port than the API/web content
 management:
   port: 8008
-  security:
-    enabled: false
```
Comment (esotrope) [inline admin-service/src/main/resources/application.yml]: as discussed in previous PR this config is overridden when a `@EnableWebSecurity` annotated file is in your app

File: report-processor/src/main/resources/application.yml [hunk 2]
```diff
@@ -2,27 +2,14 @@
...
-  security:
-    enabled: false
-
-  ### this can be removed when spring-session dep is moved from common-web to webapp
-  health:
-    redis:
-      enabled: false
-
-server:
-  port: 8080
```
Comment (esotrope) [inline report-processor/src/main/resources/application.yml]: this is the default already

File: reporting-service/src/main/java/org/opentestsystem/rdw/reporting/ApplicationProperties.java [hunk 3]
```diff
@@ -15,7 +15,6 @@
     private IrisProperties iris = new IrisProperties();
     private AnalyticsProperties analytics = new AnalyticsProperties();
     private ReportServiceProperties reportservice = new ReportServiceProperties();
-    private String adminWebappUrl;
```
Comment (esotrope) [inline reporting-service/src/main/java/org/opentestsystem/rdw/reporting/ApplicationProperties.java]: no longer needed now the admin is internalized

File: olap/src/main/resources/application.yml [hunk 4]
```diff
@@ -2,8 +2,6 @@
 # since those are the only end-points, switch to a less standard port using the main server.port config
```
Comment (malaffoon) [inline olap/src/main/resources/application.yml]: nit: this comment should probably be changed to be the same as from the file above

File: report-processor/src/main/resources/application.yml [hunk 5]
```diff
@@ -2,27 +2,14 @@
 # since those are the only end-points, switch to a less standard port using the main server.port config
```
Comment (malaffoon) [inline report-processor/src/main/resources/application.yml]: nit: ditto

File: reporting-service/src/main/resources/application.yml [hunk 6]
```diff
@@ -77,9 +72,6 @@ app:
   analytics:
     trackingId: ${rdw.reporting.analytics.trackingId:}
 
-  # TODO can go away when admin-webapp joins webapp
-  admin-webapp-url:
-
   external-links:
```
Comment (malaffoon) [inline reporting-service/src/main/resources/application.yml]: Are these delivered up as part of app properties as well? Feels like they might belong in that JIRA that moves things around.

Comment (esotrope) [reply reporting-service/src/main/resources/application.yml]: yes my running list is thus:
```yml
app:
  # TODO system prop
  state:
    code: CA
    name: California

  # TODO system/webapp prop
  iris:
    vendorId: ${rdw.reporting.iris.vendorId}

  # TODO system/webapp prop
  analytics:
    trackingId: ${rdw.reporting.analytics.trackingId:}

  # TODO webapp props
  external-links:
    interpretive-guide: https://portal.smarterbalanced.org/library/en/reporting-system-interpretive-guide.pdf
    user-guide: https://portal.smarterbalanced.org/library/en/reporting-system-user-guide.pdf

  # TODO system prop
  min-item-data-year: 2016
```

**Comment by esotrope** [comment]
@malaffoon ill ping you when the code builds
Going to merge now to keep these refactors rolling

**Pull request merged**